Formula/wxpython.rb

```diff
@@ -29,9 +29,9 @@ class Wxpython < Formula
     depends_on :python
     depends_on FrameworkPythonRequirement
   end
-  depends_on "wxmac"
 
   option :universal
+  depends_on "wxmac"
```

Comment:
Need an empty line between the last option & first `depends_on`. You can move the other block with `depends_on` within as well:

```ruby
option :universal

if MacOS.version <= :snow_leopard
  depends_on :python
  depends_on FrameworkPythonRequirement
end
depends_on "wxmac"
```